### ISSUE TITLE
`pod trunk push` failed!

### ISSUE THREAD

**Issue by nixzhu** [CocoaPods/CocoaPods#5838]
I have updated my framework for Swift 2.3, then try `pod trunk push`.

Is there any thing wrong? What's the `App.xcworkspace`?

```
$ pod --version
1.1.0.beta.2
```

```
$ cat MonkeyKing.podspec 
Pod::Spec.new do |s|

  s.name        = "MonkeyKing"
  s.version     = "0.9.4"
  s.summary     = "MonkeyKing helps you to post messages to Chinese Social Networks, or do OAuth."

  s.description = <<-DESC
                   You just want to share some information to WeChat, QQ, ...
                   Why use their buggy SDKs?
                   DESC

  s.homepage    = "https://github.com/nixzhu/MonkeyKing"

  s.license     = { :type => "MIT", :file => "LICENSE" }

  s.authors           = { "nixzhu" => "zhuhongxu@gmail.com" }
  s.social_media_url  = "https://twitter.com/nixzhu"

  s.ios.deployment_target   = "8.0"

  s.source          = { :git => "https://github.com/nixzhu/MonkeyKing.git", :tag => s.version }
  s.source_files    = "Sources/*.swift"
  s.requires_arc    = true

end
```

```
$ pod trunk push --verbose

[!] Found podspec `MonkeyKing.podspec`

Updating spec repo `master`
Validating podspec
    MonkeyKing (0.9.4) - Analyzing on iOS 8.0 platform.
  Preparing

Analyzing dependencies

Inspecting targets to integrate
  Using `ARCHS` setting to build architectures of target `Pods-App`: (``)

Fetching external sources
-> Fetching podspec for `MonkeyKing` from `/Users/nix/Documents/Frameworks/MonkeyKing/MonkeyKing.podspec`

Resolving dependencies of 

Comparing resolved specification to the sandbox manifest
  A MonkeyKing

Downloading dependencies

-> Installing MonkeyKing (0.9.4)
  > Copying MonkeyKing from `/Users/nix/Library/Caches/CocoaPods/Pods/External/MonkeyKing/7d3c2086f3de31c8c16ad817d010539f-19bea` to
  `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/MonkeyKing`
  - Running pre install hooks

Generating Pods project
  - Creating Pods project
  - Adding source files to Pods project
  - Adding frameworks to Pods project
  - Adding libraries to Pods project
  - Adding resources to Pods project
  - Linking headers
  - Installing targets
    - Installing target `MonkeyKing` iOS 8.0
      - Generating Info.plist file at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/MonkeyKing/Info.plist`
      - Generating module map file at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/MonkeyKing/MonkeyKing.modulemap`
      - Generating umbrella header at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/MonkeyKing/MonkeyKing-umbrella.h`
    - Installing target `Pods-App` iOS 8.0
      - Generating Info.plist file at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/Pods-App/Info.plist`
      - Generating module map file at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/Pods-App/Pods-App.modulemap`
      - Generating umbrella header at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/Pods-App/Pods-App-umbrella.h`
  - Running post install hooks
  - Writing Xcode project file to `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Pods.xcodeproj`
  - Writing Lockfile in `../../../../../var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Podfile.lock`
  - Writing Manifest in `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Manifest.lock`

Integrating client project

[!] Please close any current Xcode sessions and use `App.xcworkspace` for this project from now on.

Integrating target `Pods-App` (`../../../../../var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/App.xcodeproj` project)
  Adding Build Phase '[CP] Embed Pods Frameworks' to project.
  Adding Build Phase '[CP] Copy Pods Resources' to project.
  Adding Build Phase '[CP] Check Pods Manifest.lock' to project.
  - Running post install hooks
  Pod installation complete! There is 1 dependency from the Podfile and 1 total pod installed.
   Building with xcodebuild. 
xcodebuild clean build -workspace App.xcworkspace -scheme App -configuration Release CODE_SIGN_IDENTITY=- -sdk iphonesimulator -destination id=790C7FDF-40BA-4EB2-90D7-7223E4443329
Build settings from command line:
    CODE_SIGN_IDENTITY = -
    SDKROOT = iphonesimulator10.0

xcodebuild: error: 'App.xcworkspace' does not exist.
 -> MonkeyKing (0.9.4)
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code.
    - NOTE  | [iOS] xcodebuild:  xcodebuild: error: 'App.xcworkspace' does not exist.

[!] The spec did not pass validation, due to 1 error.

/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-trunk-1.0.0/lib/pod/command/trunk/push.rb:111:in `validate_podspec'
/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-trunk-1.0.0/lib/pod/command/trunk/push.rb:61:in `run'
/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/claide-1.0.0/lib/claide/command.rb:334:in `run'
/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0.beta.2/lib/cocoapods/command.rb:50:in `run'
/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0.beta.2/bin/pod:55:in `<top (required)>'
/Users/nix/.rvm/rubies/ruby-2.3.0/bin/pod:23:in `load'
/Users/nix/.rvm/rubies/ruby-2.3.0/bin/pod:23:in `<main>'
/Users/nix/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `eval'
/Users/nix/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by benasher44** [comment]
This is a dup of #5805. Closing.

**Comment by benasher44** [comment]
This will be fixed in the upcoming release.

**Comment by nixzhu** [comment]
with cocoapods-core-1.1.0.rc.1, failed again.

```
$ pod trunk push --verbose

[!] Found podspec `MonkeyKing.podspec`

Updating spec repo `master`
Validating podspec
    MonkeyKing (0.9.4) - Analyzing on iOS 8.0 platform.
  Preparing

Analyzing dependencies

Inspecting targets to integrate
  Using `ARCHS` setting to build architectures of target `Pods-App`: (``)

Fetching external sources
-> Fetching podspec for `MonkeyKing` from `/Users/nix/Documents/Frameworks/MonkeyKing/MonkeyKing.podspec`

Resolving dependencies of 

Comparing resolved specification to the sandbox manifest
  A MonkeyKing

Downloading dependencies

-> Installing MonkeyKing (0.9.4)
 > Git download
 > Git download
     $ /Applications/Xcode.app/Contents/Developer/usr/bin/git clone https://github.com/nixzhu/MonkeyKing.git
     /var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/d20160911-26080-1hb5voo --template= --single-branch --depth 1 --branch 0.9.4
     Cloning into '/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/d20160911-26080-1hb5voo'...
     Note: checking out 'fc96bc33868e2a3b07eee7919f287a5fa35b6fb8'.
     
     You are in 'detached HEAD' state. You can look around, make experimental
     changes and commit them, and you can discard any commits you make in this
     state without impacting any branches by performing another checkout.
     
     If you want to create a new branch to retain commits you create, you may
     do so (now or later) by using -b with the checkout command again. Example:
     
       git checkout -b <new-branch-name>
     
  > Copying MonkeyKing from `/Users/nix/Library/Caches/CocoaPods/Pods/External/MonkeyKing/7d3c2086f3de31c8c16ad817d010539f-19bea` to
  `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/MonkeyKing`
  - Running pre install hooks

Generating Pods project
  - Creating Pods project
  - Adding source files to Pods project
  - Adding frameworks to Pods project
  - Adding libraries to Pods project
  - Adding resources to Pods project
  - Linking headers
  - Installing targets
    - Installing target `MonkeyKing` iOS 8.0
      - Generating Info.plist file at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/MonkeyKing/Info.plist`
      - Generating module map file at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/MonkeyKing/MonkeyKing.modulemap`
      - Generating umbrella header at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/MonkeyKing/MonkeyKing-umbrella.h`
    - Installing target `Pods-App` iOS 8.0
      - Generating Info.plist file at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/Pods-App/Info.plist`
      - Generating module map file at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/Pods-App/Pods-App.modulemap`
      - Generating umbrella header at `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Target Support
      Files/Pods-App/Pods-App-umbrella.h`
  - Running post install hooks
  - Writing Xcode project file to `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Pods.xcodeproj`
  - Writing Lockfile in `../../../../../var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Podfile.lock`
  - Writing Manifest in `../../../../../private/var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/Pods/Manifest.lock`

Integrating client project

[!] Please close any current Xcode sessions and use `App.xcworkspace` for this project from now on.

Integrating target `Pods-App` (`../../../../../var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/App.xcodeproj` project)
  Adding Build Phase '[CP] Embed Pods Frameworks' to project.
  Adding Build Phase '[CP] Copy Pods Resources' to project.
  Adding Build Phase '[CP] Check Pods Manifest.lock' to project.
  - Running post install hooks
  Pod installation complete! There is 1 dependency from the Podfile and 1 total pod installed.
   Building with xcodebuild. 
xcodebuild clean build -workspace /var/folders/40/qkp56d0d7y500_t83ry6hbt40000gn/T/CocoaPods/Lint/App.xcworkspace -scheme App -configuration Release CODE_SIGN_IDENTITY=- -sdk iphonesimulator -destination id=790C7FDF-40BA-4EB2-90D7-7223E4443329
Build settings from command line:
    CODE_SIGN_IDENTITY = -
    SDKROOT = iphonesimulator10.0

=== CLEAN TARGET MonkeyKing OF PROJECT Pods WITH CONFIGURATION Release ===

Check dependencies
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

** CLEAN FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
=== BUILD TARGET MonkeyKing OF PROJECT Pods WITH CONFIGURATION Release ===

Check dependencies
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
 -> MonkeyKing (0.9.4)
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code.

[!] The spec did not pass validation, due to 1 error.

/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-trunk-1.0.0/lib/pod/command/trunk/push.rb:111:in `validate_podspec'
/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-trunk-1.0.0/lib/pod/command/trunk/push.rb:61:in `run'
/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/claide-1.0.0/lib/claide/command.rb:334:in `run'
/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/command.rb:50:in `run'
/Users/nix/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0.rc.1/bin/pod:55:in `<top (required)>'
/Users/nix/.rvm/rubies/ruby-2.3.0/bin/pod:23:in `load'
/Users/nix/.rvm/rubies/ruby-2.3.0/bin/pod:23:in `<main>'
/Users/nix/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `eval'
/Users/nix/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by iNoles** [comment]
@nixzhu did you enabled "Use Legacy Swift Language Version?"

**Comment by nixzhu** [comment]
@iNoles Yes, you can check it at https://github.com/nixzhu/MonkeyKing if you like.

**Comment by yoichitgy** [comment]
This workaround worked in my case with `pod --version` 1.1.0.rc.1
https://github.com/CocoaPods/CocoaPods/issues/5796#issuecomment-245803734

(I couldn't find a correct way to specify `SWIFT_VERSION` in a podspec.)

**Comment by benasher44** [comment]
The second issue you ran into will be fixed by #5841

**Comment by benasher44** [comment]
This will be fixed in 1.1.0.rc.2 (#5841).
